lib/logger.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const {createLogger, format, transports} = require('winston');
+
+function getDatadogDomain(region) {
+    if (region === 'eu') {
+        return 'http-intake.logs.datadoghq.eu';
+    } else if (region === 'us3') {
+        return 'http-intake.logs.us3.datadoghq.com';
+    } else if (region === 'us5') {
+        return 'http-intake.logs.us5.datadoghq.com';
+
+    }
+    return 'http-intake.logs.datadoghq.com';
+}
+
+function dataDogTransportOptions(apiKey, tags, region, source, service) {
+    const ddtags = encodeURI(tags);
+    if (!source) {
+        source = 'nodejs';
+    }
+    return {
+        host: getDatadogDomain(region),
+        path: `/api/v2/logs?dd-api-key=${apiKey}&ddsource=${source}&service=${service}&ddtags=${ddtags}`,
+        ssl: true,
+        format: format.json(),
+    };
+}
+
+module.exports = function (opts = {}) {
+    if (!opts.apiKey) {
+        throw new Error('Missing required option: `apiKey`')
+    }
+    if (!opts.service) {
+        throw new Error('Missing required option: `service`')
+    }
+
+    let additionalTransports = [];
+    if (opts.additionalTransports) {
+        additionalTransports = opts.additionalTransports;
+        delete opts.additionalTransports;
+    }
+
+    return createLogger({
+        level: 'info',
```

Comment:
do we want to make this configurable?

Reply:
It is, this is the default. 

opts passed in after the defaults are set

Reply:
This line here allows the override: https://github.com/gymshark/datadog-client-js/pull/3/files#diff-9c6fae252a0db7a8d112dee362cbc6b40443ab013ad80bc29b233406cf6fd546R53